### ISSUE TITLE
iOS 8 Notification Widget

### ISSUE THREAD

**Issue by chrissearle** [javaBin/ems-ios#148]
Just a cool nice to have - your next starred session in room ... or similar

**Comment by ronnienessa** [comment]
Agree – we need to find some time working on this soon...

**Comment by chrissearle** [comment]
Right - adding a widget (or today extension) will add a new target. This will target sdk 8. But - it will be embedded in the app - and the app can still target 7. If you install an app with an embedded extension on pre-8 iOS then the extension just gets ignored.

**Comment by ronnienessa** [comment]
Given our resource constraint, does it make sense to keep supporting iOS 7? iOS 7 user can continue running current version.

**Comment by chrissearle** [comment]
Depends. So far all I've found is the notification registration (which I think is complete) - so - if the level of work for keeping 7 is that low - why bother changing it? However - at the point where we actually have to work to maintain it - then sure - junk 7 - after all - that's what I've done every year so far - only supported the latest ;)

**Comment by ronnienessa** [comment]
One reason is to get rid of that ugly auto-cell resizing to support different font sizes. It's giving me a headache. 

Vennlig hilsen 
Webstep AS

Ronnie Nessa | Seniorkonsulent
Tlf: +47 970 69 512 | ronnie.nessa@webstep.no

**Comment by chrissearle** [comment]
Fair enough :) I'm not too worried - it is after all a technology conference - some level of up to date we can expect. 8 will be nearly a year old at that point.
